gpkit/substitution.py

```diff
@@ -70,7 +70,40 @@ def vectorsub(subs, var, sub, varset):
         subs[var] = sub
 
 
-def substitution(varlocs, varkeys, exps, cs, substitutions, val=None):
+def separatesubs(nomial, substitutions):
```

Comment:
hmm, this seems to mostly do things `if is_sweepvar(sub)`. should it be renamed?